bcos-executor/src/executor/TransactionExecutor.cpp

```diff
@@ -2467,6 +2467,7 @@ std::unique_ptr<protocol::ExecutionMessage> TransactionExecutor::toExecutionResu
     message->setSeq(params->seq);
     message->setOrigin(std::move(params->origin));
     message->setGasAvailable(params->gas);
+    message->setEffectiveGasPrice(boost::lexical_cast<std::string>(params->effectiveGasPrice));
```

Comment:
Is effectiveGasPrice a decimal or hexadecimal string?

Reply:
It is a hexadecimal string.